splunk_connector.py

```diff
@@ -1270,7 +1267,7 @@ def _run_query(self, search_query, action_result, attach_result=False, kwargs_cr
         # Create the job
         for attempt_count in range(0, RETRY_LIMIT):
             try:
-                job = self._service.jobs.create(search_query, **kwargs_create)
+                job: splunk_client.Job = self._service.jobs.create(search_query, **kwargs_create)
```

Comment:
can we do the same generalization as for `_wait_until_splunk_job_results_are_ready` but for `jobs.create`?

Reply:
Yeah, we can, good idea